psi4/src/psi4/libscf_solver/rhf.cc

```diff
@@ -286,8 +285,19 @@ void RHF::form_F() {
     }
 }
 
-void RHF::form_C() {
-    diagonalize_F(Fa_, Ca_, epsilon_a_);
+void RHF::form_C(double shift) {
```

Comment:
One detail worth double-checking: does Psi define the RHF Fock matrix as `F_a + F_b` or `(F_a + F_b) / 2`? If the former, we should double the shift factor as well.

Reply:
The former choice would give the wrong orbital energies, so I don't think that's an issue.